packages/server/src/main/java/software/uncharted/terarium/hmiserver/TerariumApplication.java

```diff
@@ -13,50 +13,42 @@
 import org.springframework.context.annotation.PropertySource;
 import org.springframework.scheduling.annotation.EnableScheduling;
 
-import java.util.HashSet;
-import java.util.Set;
+import software.uncharted.terarium.hmiserver.configuration.Config;
 
 @SpringBootApplication
 @EnableFeignClients
 @EnableScheduling
 @PropertySource("classpath:application.properties")
 public class TerariumApplication {
+
+	@Autowired
```

Comment:
Not a deal breaker but in general we're now trying to avoid `@Autowired` in favour of a `@RequiredArgsConstructor`